src/shipraT/SecondTohour.java

```diff
@@ -0,0 +1,23 @@
+package shipraT;
+
+import java.util.Scanner;
+
+public class SecondTohour {
+	public static void main(String[] args) {
+		int second =0;
+		int minitues =0;
+		int hour =0;
+		
+	Scanner sc = new Scanner(System.in);	//Scanner class to read value
+	
+		System.out.println("how many seconds ?");
+			second = sc.nextInt();
+			  if(second>=60) {
```

Comment:
Check output for 130 second 
second(time) input and output will print same.
think on this.
130 seconds
0 hr 2 min and 30 seconds output expected